### PR TITLE
fixes issues #3,4,6,7,8

### PR DESCRIPTION
**Pull request by siyaoli7**
fixes issues #3,4,6,7,8

### CODE REVIEW

File: src/PolynomialDerivative.java~ [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import java.util.Scanner;
+
```
Comment (jaeaster) [inline src/PolynomialDerivative.java~]: Remove this file from git and add `*~` to your .gitignore file

File: javadoc/allclasses-frame.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<!-- NewPage -->
```
Comment (jaeaster) [inline javadoc/allclasses-frame.html:2]: add `javadoc` to your .gitignore and make sure these files aren't tracked by git

File: README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+# lab02-siyaoli7
```
Comment (jaeaster) [inline README.md]: Change this to reflect a short description of the project

File: src/Polynomial.java [hunk 4]
```diff
@@ -0,0 +1,674 @@
...
+	and for testing plus, minus and times.
+
+	At Unix command line, use single quotes to make the entire
+	Polynomial be a single argument, and use 
+	\* when operating is * (to avoid
+	having * expanded as a wildcard.)  For example:
+	<code>java -cp build Polynomial 'x^2 + 2x + 3' \* 'x - 4'</code>
+     */
+
+    public static void main (String [] args) {
```
Comment (jaeaster) [inline src/Polynomial.java:616]: This class should not contain a main function as PolynomialDerivative contains the main

File: src/PolynomialDerivative.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+/** <p>PolynomialDerivative calculate the derivative of input polynomial</p>
+
+    <p>PolynomialDerivative allows users input data from console or command line</p>
+
+    @author Siyao Li
+    @version UCSB, CS56, F17
+*/
+
+public class PolynomialDerivative{
```
Comment (jaeaster) [inline src/PolynomialDerivative.java]: This class should extend `Polynomial` and include the implementation of the `derivative()` method

**Review by jaeaster — CHANGES_REQUESTED** [review]
Please fix the issues referenced in my comments.

Also this pull request does not include any gradle configuration, therefore I don't believe it fixes issues #3. Also I don't see any new tests so #8 is not addressed either

**Comment by jaeaster** [comment]
See here for example build.gradle

https://github.com/ehenderson1698/cs56-networking-expression-evaluator/blob/master/build.gradle

**Comment by siyaoli7** [comment]
The derivative() will return a PolynomialDerivative object now.
I left javadoc in repo for issue #1

**Pull request merged**